### DISCUSSION TITLE
Is SolidJS faster than React?

### DISCUSSION THREAD

**Discussion by Sachin066** [community/community#67730] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The SolidJS development team claims

**Comment by Sachin066** [comment]
its reactivity engine is faster than other popular frameworks like React and Vue

**Comment by Rinki2002** [comment]
its reactivity engine is faster than other popular frameworks like React and Vue

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
